example-problems/class-removed-from-google-http-client/README.md

```diff
@@ -0,0 +1,30 @@
+This project is to demonstrate a hidden incompatibility between
+`com.google.api-client:google-api-client:1.27.0` and
+`com.google.cloud:google-cloud-bigtable:0.81.0-alpha`.
+
+Maven artifact `com.google.api-client:google-api-client:1.27.0` contains `GoogleApacheHttpTransport`
+class.
+`GoogleApacheHttpTransport.newTrustedTransport()` returns `ApacheHttpTransport` class.
+This class is available in `com.google.http-client:google-http-client:1.27.0`, which is one of the
+transitive dependencies of the google-api-client artifact.
+
+However, when the `pom.xml` uses `com.google.api-client:google-api-client:1.27.0` and
+`com.google.cloud:google-cloud-bigtable:0.81.0-alpha` together in this order, Maven picks up 
+`com.google.http-client:google-http-client:1.28.0` in the transitive dependencies.
```

Comment:
That shouldn't happen. What you describe should pick up com.google.api-client:google-api-client:1.27.0 and not need the transitive dependencies.